### ISSUE TITLE
Unattended installation ended with error

### ISSUE THREAD

**Issue by serock** [agama-project/agama#1739]
I tried an unattended installation of Leap 16.0 using the `agama-installer.x86_64-10.0.0-openSUSE-Build27.7.iso` in VirtualBox 7.0.18 on Leap 15.6. At the end of the installation, an error was displayed:
![Screenshot from 2024-11-08 03-10-39](https://github.com/user-attachments/assets/ac3a579d-32ed-4eff-8442-dd6c6e5b79c2)

The effective [profile.json](https://github.com/user-attachments/files/17677228/profile.json) is attached with the user and password info omitted.

The end of the `agama.out.log` had:

```
Nov 08 09:08:01 agama agamactl[3007]: [INFO]: software: Commit result [688, [], [], [], []]
Nov 08 09:08:01 agama agamactl[2991]: [INFO]: manager: Configure the system (3/3)
Nov 08 09:08:02 agama agamactl[39077]: /mnt: /etc/locale.conf written.
Nov 08 09:08:02 agama agamactl[39077]: /mnt: /etc/vconsole.conf written.
Nov 08 09:08:02 agama agamactl[39077]: /mnt: /etc/localtime written
Nov 08 09:08:15 agama agamactl[2989]: Loading mocked module /usr/lib64/ruby/gems/3.3.0/gems/agama-yast-10.devel407/lib/agama/dbus/y2dir/modules/InstFunctions.rb
Nov 08 09:08:20 agama agamactl[2991]: [INFO]: manager: Install phase done
```

The installation of Leap 16.0 seemed to be successful; I was able to reboot the virtual machine and login to Leap 16.0.

**Comment by imobachgs** [comment]
Thanks for the report. The installation was finished successfully, but the web UI crashed. We will try to reproduce the problem.

**Comment by imobachgs** [comment]
I was able to reproduce the problem using [this profile](
[bug.json](https://github.com/user-attachments/files/17703254/bug.json). It seems to related to the storage layer, as I see two failing requests to `api/storage/proposal/settings` and `api/storage/proposal/actions`.

I am attaching some debugging information:

* [Agama logs](https://github.com/user-attachments/files/17703277/agama-logs.tar.gz)
* [Browser console logs](https://github.com/user-attachments/files/17703278/console-export-2024-11-11_14-44-20.txt)

**Comment by imobachgs** [comment]
Actually, if you remove the encryption password from `bug.json` and try to import that profile (from the CLI), the storage proposal will fail and the UI won't handle it properly.

**Comment by joseivanlopez** [comment]
I haven't checked this yet. BTW, removing the encryption password should report an error because the password is required by the schema. Is Agama validating the schema while importing the profile?

**Comment by imobachgs** [comment]
Sorry, I did not explain it well. It is not removing but setting the password to an empty string (which is "valid"). However, in my latest test I was unable to reproduce that problem.

However, the original issue still applies.

**Comment by teclator** [comment]
Should be fixed by #1793, thanks for the report!
